packages/react-components/src/components/styles/InputBoxComponent.style.ts

```diff
@@ -62,15 +62,16 @@ export const textFieldStyle = (errorColor: string, hasErrorMessage: boolean, dis
       fontSize: '8.25rem'
     },
     wrapper: {},
+    /**
+     * Border of the field group is set to none, there is a wrapper div that encloses text field and
+     * uploaded files.
+     */
     fieldGroup: {
+      outline: 'none',
+      border: 'none',
       height: 'auto',
       minHeight: '0',
-      borderRadius: '0.25rem',
-      borderColor: borderColor,
-      borderWidth: disabled ? '0px' : '1px',
-      ':hover': { borderColor: borderColor },
-      ':active': { borderColor: borderColor },
-      ':after': { borderColor: borderColor, borderRadius: '0.25rem' }
+      ':after': { border: 'none' }
```

Comment:
Can we add a comment why are are adding border none here?

Reply:
done

Reply:
Sorry I don't understand this comment.
Can we simplify it?
Something like
`Removing fieldGroup border to make sure no border is rendered around the text area in sendbox`